src/ui/public/utils/as_pretty_string.js

```diff
@@ -0,0 +1,13 @@
+/**
+ * Convert a value to a presentable string
+ * @param  {any} val - the value to transform
+ * @return {string}
+ */
+export function asPrettyString(val) {
```

Comment:
Would be nice to have some unit tests for this and shorten_dotted_string.js below.